src/components/DoiMetadata/DoiMetadata.tsx

```diff
@@ -131,6 +125,61 @@ const DoiMetadata: React.FunctionComponent<Props> = ({item}) => {
     )
   }
 
+  const registered = () => {
+    return (
+      <div className="registered">
+      DOI registered
+      {item.registered &&
+        <span> {new Date(item.registered).toLocaleDateString('en-US', { year: 'numeric', month: 'long', day: 'numeric' })}</span>
+      } via {item.registrationAgency.name}.
+      </div>
+    )
+  }
+
+  const tooltipResourceTypeGeneral = (
+    <Tooltip id="tooltipResourceTypeGeneral">
+      The general type of the content.
+    </Tooltip>
+  )
+
+  const tooltipFieldsOfScience = (
+    <Tooltip id="tooltipFieldsOfScience">
+      The OECD Fields of Science for the content.
+    </Tooltip>
+  )
+
+  const tooltipLanguage = (
+    <Tooltip id="tooltipLanguage">
+      The primary language of the content.
+    </Tooltip>
+  )
+
+  const tags = () => {
+    return (
+      <div className="tags">
+        {item.types.resourceTypeGeneral &&
```

Comment:
The general type is already visible in other places in the UI, Why do we need to duplicate this information without a clear use case.

Reply:
This is the only place where we show resourceTypeGeneral in the UI. Showing it after the title has been removed. I wanted to align the tags with the facets where it makes sense.